tests/test_file.py

```diff
@@ -0,0 +1,207 @@
+"""Test file reader."""
+import sys
+from tempfile import TemporaryDirectory
+from typing import Any, Dict
+
+sys.path.append("..")
```

Comment:
tbh i'm not a huge fan of having to add this to every test but also can't think of a better solution atm (an alternative would be to reintroduce a python package). so ok with this.

we could make this more explicit though:
```
sys.path.append(Path(__file__).parent.parent)
```

Reply:
I considered reintroducing the package but it didn't seem like the right choice just for the sake of tests. Having the package without the intention of actually releasing LlamaHub as a package (which I think is not worth the effort to maintain rn) could introduce confusion and surface area for errors (e.g. people importing stuff incorrectly from within loaders).

I'll make that change though